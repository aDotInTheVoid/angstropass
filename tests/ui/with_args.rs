#[nanopass::langs(no no no)]
mod foo {}

fn main() {}
