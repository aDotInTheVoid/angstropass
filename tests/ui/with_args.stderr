error: this attribute does not take any arguments
 --> tests/ui/with_args.rs:1:19
  |
1 | #[nanopass::langs(no no no)]
  |                   ^^^^^^^^
